src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsGptPageParamOasis.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.driverprovider.UseUnstablePageLoadStrategy;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEng
+ */
+public class TestAdsGptPageParamOasis extends TemplateNoFirstLoad {
```

Comment:
Yeah. As we discussed: I don't see any reason why we couldn't re-use this test for Mercury as well. But let's leave it as-is and rename it once we really use it for Mercury too ;)